ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -237,6 +237,8 @@ mac_address_table:
 
         # Dictionary of SVIs | Required.
         # This will create both the L3 SVI and L2 VLAN based on filters applied to l3leaf and l2leaf.
+        # Any SVI setting, defined under svis[svi] can also be defined under the svis[svi].nodes[node]
```

Comment:
Please include this comment above the nodes section as well. We could consider (re)moving the specific keys under nodes, since we now support any key.